src/styles/partials/desktop/_recapitulatif.scss

```diff
@@ -0,0 +1,42 @@
+.recapitulatif-form {
+  .chapter-block:not(:last-child) {
+    margin-bottom: 1.25rem;
+  }
+
+  .subtitle {
+    font-weight: 700;
+  }
+
+  .recapitulatif-row {
+    $edit-col: 70px;
+    $question-col: calc(70% - #{$edit-col});
+    $value-col: calc(30% - #{$edit-col});
+
+    display: flex;
+    justify-content: space-between;
+
+    &:not(:last-child) {
+      margin-bottom: 10px;
```

Comment:
using multiple of 8px in margin and padding is a good practice

Reply:
Intéressant. Je peux savoir d'où vient cette règle ? :)